<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1185</width>
    <height>713</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Video Downloader</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>1111</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>Add Job</string>
    </property>
    <widget class="QLineEdit" name="lineEditURL">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>50</y>
       <width>531</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btnAddJob">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>50</y>
       <width>171</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Download</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnSetting">
     <property name="geometry">
      <rect>
       <x>902</x>
       <y>50</y>
       <width>161</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Setting</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_info">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>90</y>
       <width>531</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QTableWidget" name="downloadJobList">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>220</y>
      <width>1111</width>
      <height>491</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_info_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>180</y>
      <width>531</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Right click on task for more actions</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1185</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
